spdx-utils/src/main/kotlin/model/SpdxDocument.kt

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.spdx.model
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonProperty
+
+private const val SPDX_VERSION = "SPDX-2.2"
+private const val DATA_LICENSE = "CC0-1.0"
+
+/**
+ * An SPDX document as specified by https://github.com/spdx/spdx-spec/tree/master/chapters and
+ * https://github.com/spdx/spdx-spec/blob/development/v2.2.1/examples/.
```

Comment:
Better use a tag instead of a branch name here.

Reply:
There is not tag for 2.2.1 yet as it is still subject to change IIUC.

Reply:
I acknowledge that, but I yet believe using a tag is better than using a branch here. Does the difference between 2.2 and 2.2.1 actually matter in this context?

Reply:
Hm, if the implementation is made against specification version 2.2.1 linking against 2.2 is misleading IMO no matter how big the differences are. Why do you think it is better to use that 2.2. tag?

Reply:
The difference between 2.2. and 2.2.1 is 1) spec's layout is being rewritten in the ISO standard and 2) several fixes in the YAML and JSON examples

Reply:
I've added the SHA1 and also fixed the above link which pointed to master.

Reply:
> Why do you think it is better to use that 2.2. tag?

Because tag contents, in contrast to branch contents, do not change, and that makes it crystal clear what the state of the specs was when this was implemented.

> The difference between 2.2. and 2.2.1 is [...]

That sounds to me as if it's safe for us to use the 2.2 tag, as the 2.2.1 changes to not affect the implementation of the model.

Reply:
The SHA1 I've added addresses the points you mention IIUC.